lib/ansible/modules/files/acl.py

```diff
@@ -73,6 +73,14 @@
     type: bool
     default: 'no'
     version_added: "2.0"
+
+  nomask:
```

Comment:
instead of the negative, make it `set_mask` and default to 'yes', same functionality but doesn't require people inverting the feature to use it.

Reply:
I thought about that but, confusingly enough, setfacl also has a --mask option whose behaviour is not the opposite of --no-mask and the default is different from both, so I thought that sticking to the original flag names would be less surprising to the users.